### DISCUSSION TITLE
User Data List Source

### DISCUSSION THREAD

**Discussion by enkelmedia** [leekelleher/umbraco-contentment#136] [Question] [answered]
I saw [a question on Our](https://our.umbraco.com/forum/using-umbraco-and-getting-started/107196-can-a-multi-user-picker-be-created-in-umbraco-8#comment-333683) someone wanted a “User Picker” that supports multiple users. 

My first idea was Contentment :) But does not look like Umbraco Users is a built in data source?

Just posted the idea 💡

**Comment by leekelleher** [comment, marked as answer]
Added a data-source for Umbraco Users: 8691c4407fe28f1e9801aa7f3d9d9f27dec0476b
I also did an Umbraco User Groups data-source too: 8691c4407fe28f1e9801aa7f3d9d9f27dec0476b

They'll be part of the upcoming 2.2.0 release.

**Comment by enkelmedia** [comment]
Awesome! I wanted to post the idea and figured I could help out if you liked the idea. Days later you’ve fixed it :) Great man!

**Comment by leekelleher** [comment]
...and it's been released in [Contentment v2.2.0](https://github.com/leekelleher/umbraco-contentment/releases/tag/2.2.0)! 🌟
